### ISSUE TITLE
Precise-engine giving error, failed to load parameter

### ISSUE THREAD

**Issue by gourabdg47** [MycroftAI/mycroft-precise#119]
Hi, am getting this error everytime I try to load pb file with precious-engine 'Failed to load parameter from cough.cv.pb.params', it only works for the hey-microft.pb file rest (cough.cv.pb, my-computer.pb) gives the above error , I even trained with my own dataset  but still get the same error. I followed all the given steps for training. This is what am getting:
Traceback (most recent call last):
  File "precise/scripts/stream.py", line 52, in <module>
  File "precise/scripts/stream.py", line 40, in main
  File "precise/network_runner.py", line 79, in __init__
    return self.model.predict(inputs)
  File "precise/network_runner.py", line 30, in __init__
    def predict(self, inputs: np.ndarray) -> np.ndarray:
  File "precise/network_runner.py", line 44, in load_graph  
  File "tensorflow/python/util/deprecation.py", line 316, in new_func
  File "tensorflow/python/framework/importer.py", line 541, in import_graph_def
ValueError: No op named ClipByValue in defined operations.
[7451] Failed to execute script stream

Thank you

**Comment by el-tocino** [comment]
Did you update precise after building the model?  What version of precise are you on?

**Comment by gourabdg47** [comment]
Hi, I don't know how to update precise after building the model. How do I check the precise version? I have cloned and followed the steps given in here: https://github.com/MycroftAI/mycroft-precise#source-install , after running setup.sh, I followed the steps from here: https://github.com/MycroftAI/mycroft-precise/wiki/Training-your-own-wake-word#how-to-train-your-own-wake-word , to train my dataset. I did:
* precies-train -e 100 model/file.net data/    (the data folder contain these 3 folders not-wake-word, wake-word, test)
Then i converted the .net file to .pb file using the following cmd:
* precise-convert model/file.net -o file.pb
Then I used that .pb file to run the following cmd: 
* arecord -r 16000 -c 1 -f S16_LE | precise-engine/precise-engine model/file.pb 2048 | grep --invert-match e
Here is a screenshot of the error am facing:
![2020-01-08-123107_1920x1080_scrot](https://user-images.githubusercontent.com/6232246/71957101-d2767480-3212-11ea-9ccb-62d2e5b85ff6.png)
